pkg/config/config.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"crypto/rand"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/hyperledger/fabric-protos-go/common"
+)
+
+// SignConfigUpdate signs the given configuration update with a
+// specific signer.
+func SignConfigUpdate(configUpdate *common.ConfigUpdate, signer *Signer) (*common.ConfigSignature, error) {
```

Comment:
In this context, "Signer" is confusing. Yes, it's a concrete type that implements `crypto.Signer` but it's not a Signer. What you've created with `NewSigner` is essentially a SigningIdentity. We should consider names that more closely reflect what we're dealing with.

Reply:
done